Singer.API/ClientApp/src/app/modules/core/models/careuser.model.ts

```diff
@@ -9,12 +11,60 @@ export interface PaginationDTO{
 
 export interface CareUserDTO {
    id: string;
-   name:string;
+   firstName: string;
+   lastName: string;
+   email: string;
+   userName: string;
+   birthday: Date;
+   caseNumber: string;
+   ageGroup: AgeGroup; //Maybe replace by own class?
+   isExtern: boolean;
+   hasTrajectory: boolean;
+   hasNormalDayCare: boolean;
+   hasVacationDayCare: boolean;
+   hasResources: boolean;
+}
+
+export interface UpdateCareUserDTO {
+   firstName: string;
+   lastName: string;
    email: string;
    userName: string;
    birthday: Date;
    caseNumber: string;
-   ageGroup: string; //Maybe replace by own class?
+   ageGroup: AgeGroup; //Maybe replace by own class?
```

Comment:
Is this change not the implementation of our own class? If so, please remove comment

Reply:
Yes, yet it is an enum.